sysext.just

```diff
@@ -326,12 +326,9 @@ build-erofs target:
     fi
 
     echo "🔒 Creating EROFS sysext ({{compression}})"
-    version=$(podman run --rm -ti \
-        --security-opt label=disable \
-        "{{target}}" \
-        bash -c 'source /etc/os-release ; echo -n ${OSTREE_VERSION}')
+    version=$(podman inspect "{{ target }}" --format '{{' {{ index .Config.Labels "org.opencontainers.image.version" }} '}}' | grep -Po '\K[0-9].*[0-9]')
     arch=$(uname -m | sed 's/_/-/g')
-    ${SUDO} mkfs.erofs -z{{compression}} {{name}}-${version}-${arch}.raw rootfs > /dev/null
```

Comment:
Why is this removed?

Reply:
Hum, but sometimes some files are not readable by the user as they are `root:root` and not "others" readable. polkit rules are a common case.

Reply:
Yeah, this can be added back in that case. Instead of relying on that assumption.